lib/ex_webrtc/rtp/vp8_payload.ex

```diff
@@ -0,0 +1,148 @@
+defmodule ExWebRTC.RTP.VP8Payload do
+  @moduledoc """
+  Defines VP8 payload structure stored in RTP packet payload.
+  """
```

Comment:
It would be nice to include the RFCs/specs that the `vp8_x.ex` modules were based on.